layers/poi/class.sql

```diff
@@ -46,24 +48,28 @@ RETURNS TEXT AS $$
         WHEN subclass IN ('books','library') THEN 'library'
         WHEN subclass IN ('university','college') THEN 'college'
         WHEN subclass IN ('hotel','motel','bed_and_breakfast','guest_house','hostel','chalet','alpine_hut','camp_site') THEN 'lodging'
-        WHEN subclass IN ('chocolate','confectionery') THEN 'ice_cream'
+        WHEN subclass IN ('chocolate','confectionery', 'ice_cream') THEN 'ice_cream'
         WHEN subclass IN ('post_box','post_office') THEN 'post'
-        WHEN subclass IN ('cafe') THEN 'cafe'
+        WHEN subclass IN ('cafe', 'coffee') THEN 'cafe'
         WHEN subclass IN ('school','kindergarten') THEN 'school'
-        WHEN subclass IN ('alcohol','beverages','wine') THEN 'alcohol_shop'
+        WHEN subclass IN ('alcohol','beverages','wine', 'nightclub') THEN 'alcohol_shop'
         WHEN subclass IN ('bar','nightclub') THEN 'bar'
-        WHEN subclass IN ('marina','dock') THEN 'harbor'
-        WHEN subclass IN ('car','car_repair','taxi') THEN 'car'
-        WHEN subclass IN ('hospital','nursing_home', 'doctors', 'clinic') THEN 'hospital'
+        WHEN subclass IN ('marina','dock', 'boat_rental', 'boat_sharing', 'ferry_terminal') THEN 'harbor'
+        WHEN subclass IN ('car','car_repair','taxi', 'fuel', 'parking', 'car_wash', 'car_rental', 'charging_station', 'parking_space') THEN 'car'
```

Comment:
We may want different icons for these so we should not group them : we don't want a car charging station and a taxi stop to look the same in our map.